src/pybind/mgr/dashboard/frontend/src/app/shared/components/helper/helper.component.scss

```diff
@@ -1,3 +1,7 @@
 cds-tooltip {
   margin-left: 5px;
+
+  ::ng-deep .cds--tooltip-content {
+    max-inline-size: 25rem;
```

Comment:
can we use carbon spacings ?

Reply:
this was to increase the width of the tooltip component so it can fit larger text. So not sure if we can use the spacing here.